roles/openshift_metrics/tasks/cleanup.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: remove metrics components
+  command: >
+    {{ openshift.common.client_binary }} -n '{{ project }}'
```

Comment:
The original playbook adds `--namespace=openshift-infra` (hard-coded namespace name) to all `client_binary` and `admin_binary` executions:

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/tasks/install.yml

In kubernetes they are namespaces, in openshift they are usually called projects (but the flag name is kept from `kubectl`).  I don't have any strong feelings for either.